_delphi_utils_python/README.md

```diff
@@ -21,14 +21,22 @@ Source code can be found here:
 [https://github.com/cmu-delphi/covidcast-indicators/](https://github.com/cmu-delphi/covidcast-indicators/)
 
 ## Logger Usage
+To make our structured logging as useful as it can be, particularly within the context of how we use logs in Elastic, the event argument (typically the first unnamed arg) should be a static string (to make filtering easier), and each dynamic/varying value should be specified in an individual meaningfully- and consistently-named argument to the logger call (for use in filtering, thresholding, grouping, visualization, etc).
+
```

Comment:
This is a good way to move forward. I like the emphasis on consistency. I would add something like this to let people know that historically there might be info hidden in old arguments.

```suggestion
Note that for some older implementations, you might find historical info hidden in arguments that are no longer used currently. 

```

Reply:
i hope there isnt still variable info hidden in the `event` argument, thats what this is supposed to be fixing!